app/(main)/league/[leagueId]/entry/[entryId]/week/Week.test.tsx

```diff
@@ -40,13 +34,10 @@ jest.mock('@/context/AuthContextProvider', () => ({
 jest.mock('@/store/dataStore', () => ({
   useDataStore: jest.fn(() => ({
     currentWeek: 1,
-    getState: jest.fn(() => ({
-      currentWeek: 1,
-    })),
-    user: { id: '123', leagues: [] },
+    user: { id: '123', email: 'email@example.com', leagues: ['123'] },
+    weeklyPicks: {},
```

Comment:
Was there a reason behind moving weeklyPicks here?

Reply:
@choir27 it is a part of the Data Store so needed to be in that Mock.